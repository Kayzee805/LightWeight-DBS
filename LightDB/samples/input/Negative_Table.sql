SELECT * FROM Negatives N WHERE N.a<=0;