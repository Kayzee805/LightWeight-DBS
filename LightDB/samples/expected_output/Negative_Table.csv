-1,-101
-2,101
-3,-102
-4,103
-5,-104
0,-100